bugwarrior/db.py

```diff
@@ -451,6 +457,8 @@ def _bool_option(section, option, default):
                 tw.task_done(uuid=new_task['uuid'])
         except TaskwarriorError as e:
             log.exception("Unable to add task: %s" % e.stderr)
+        finally:
+            seen_uuids.add(new_task['uuid'])
```

Comment:
I've just realized that this doesn't really make any sense. If the first operation in the `try` block fails (e.g. I had  `taskw.exceptions.TaskwarriorError: <unprintable TaskwarriorError object>` error for one of my tasks), the finally statement cannot be executed, as `new_task` doesn't exist. Therefore, it would probably be more sane to move that statement inside the `try` block after the `new_task = ...` line.